src/interfaces/responses/twitch-response.ts

```diff
@@ -0,0 +1,19 @@
+export interface TwitchResponse {
+    id: string;
+    stream_id: string;
+    user_id: string;
+    created_at: string;
+    description: string;
```

Comment:
```suggestion
    description?: string;
```

Should we mark this as optional since some have no description? Or do they return empty string so it is okay if they are not optional?